muikku-school-data-pyramus/src/main/java/fi/otavanopisto/muikku/plugins/schooldatapyramus/rest/SystemAccessTokenProvider.java

```diff
@@ -39,20 +38,17 @@ public void init() {
     }
```

Comment:
Opted to not to add synchronized after all, as @Singleton provides concurrency in a much better fashion